omod-1.9/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_9/VisitController1_9Test.java

```diff
@@ -87,6 +90,16 @@ public void shouldCreateAVisit() throws Exception {
 		Assert.assertEquals(originalCount + 1, service.getAllVisits().size());
 	}
 	
+	@Test
+	public void shouldGetAVisitByUuid() throws Exception{
+		MockHttpServletRequest req = request(RequestMethod.GET, getURI() + "/" + getUuid());
+		SimpleObject result = deserialize(handle(req));
+		
+		Visit visit = service.getVisitByUuid(getUuid());
+		assertNotNull(result);
```

Comment:
Do we really need the above assert?

Reply:
After stepping back and thought about the assert, I realized we do not need it and I did the needful!